### DISCUSSION TITLE
chrome profile open

### DISCUSSION THREAD

**Discussion by amirbh1106** [elgatosf/elgato#152] [General]
hay can i setup for button to open chrome with spcific profile ?

**Comment by ChekTek** [comment]
Hi @amirbh1106!

This discussion board is for developers using the Stream Deck SDK, you may have better luck with this question filing a ticket with [Elgato Support](https://help.elgato.com/hc/en-us/requests/new) or talking with the community in the [Elgato Discord](https://discord.gg/7tPtNGBxkP).

You can also give BarRaiders [Advanced Launcher](https://marketplace.elgato.com/product/advanced-launcher-d9a289e4-9f61-4613-9f86-0069f5897125) plugin from Elgato Marketplace.

Best,
Zack
